feeder/feeder.py

```diff
@@ -18,12 +18,15 @@
 # Set up error handling
 import opbeat
 from opbeat.handlers.logging import OpbeatHandler
-if 'OPBEAT_SECRET' in os.environ:
+secret_token= os.environ.get('OPBEAT_SECRET')
+if not secret_token is None:
     opbeat_client = opbeat.Client(
-        app_id='c36849e44e',
-        organization_id='093c53b0da9d43c4976cd0737fe0f2b1',
-        secret_token=os.environ['OPBEAT_SECRET'])
-else: opbeat_client = None
+        app_id= 'c36849e44e',
+        organization_id= '093c53b0da9d43c4976cd0737fe0f2b1',
+        secret_token= secret_token
+       )
+else:
+    print "No Opbeat Token found! Opbeat Error handling is inactive."
```

Comment:
In my instance it works like this, because in line 39 ist continues with `if opbeat_client`.